python/lsst/ip/isr/overscan.py

```diff
@@ -336,18 +468,19 @@ def correctOverscan(self, exposure, amp, imageBBox, overscanBBox,
         residualMedian = stats.getValue(afwMath.MEDIAN)
         residualSigma = stats.getValue(afwMath.STDEVCLIP)
 
-        return pipeBase.Struct(ampOverscanModel=ampOverscanModel,
-                               overscanOverscanModel=overscanOverscanModel,
-                               overscanImage=overscanImage,
-                               overscanValue=overscanResults.overscanValue,
-
-                               overscanMean=overscanResults.overscanMean,
-                               overscanMedian=overscanResults.overscanMedian,
-                               overscanSigma=overscanResults.overscanSigma,
-                               overscanMeanResidual=residualMean,
-                               overscanMedianResidual=residualMedian,
-                               overscanSigmaResidual=residualSigma
-                               )
+        return pipeBase.Struct(
+            ampOverscanModel=ampOverscanModel,
+            overscanOverscanModel=overscanOverscanModel,
+            overscanImage=overscanImage,
+            overscanValue=overscanResults.overscanValue,
+            overscanMean=overscanResults.overscanMean,
+            overscanMedian=overscanResults.overscanMedian,
+            overscanSigma=overscanResults.overscanSigma,
+            overscanMeanResidual=residualMean,
+            overscanMedianResidual=residualMedian,
+            overscanSigmaResidual=residualSigma,
+            overscanBadRowsColumns=badRowsColumns,
```

Comment:
`overscanBadIndices`?  I understand `RowsColumns`, but having the user facing values be fully agnostic may be easier to follow?  This is pure suggestion, feel free to ignore.